### ISSUE TITLE
Release to PyPI?

### ISSUE THREAD

**Issue by htfy96** [GumTreeDiff/pythonparser#8]
It's kind of inconvenient to install this from source. Publishing this to PyPI would simplify the installation process a lot.

**Comment by tareq97-zz** [comment]
Hi @htfy96 ,
How did you install it without having code in PyPI. Have you managed to run this and gumtree together.

**Comment by jrfaller** [comment]
I agree! However I have no experience with packaging python projects, so I anybody want to give a hand, that would be awesome.

Cheers.
